packages/n3h-mock/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@holochain/n3h-mock",
+  "version": "0.0.1",
+  "description": "Centralized local mock of n3h",
+  "main": "lib/index.js",
+  "scripts": {
+    "lint": "standard --fix 'lib/**/*.js' 'bin/**/*.js'",
+    "mocha": "mocha lib/**/*.test.js bin/**/*.test.js",
+    "nyc": "nyc npm run mocha",
+    "test": "npm run lint && npm run nyc",
+    "show-coverage": "node -e \"require('opn')('coverage/lcov-report/index.html')\""
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "mocha": "^5.2.0",
+    "nyc": "^12.0.2",
+    "opn": "^5.4.0",
+    "sinon": "^7.1.1",
+    "standard": "^12.0.1"
+  },
+  "dependencies": {
+    "@nodeutils/defaults-deep": "^1.1.0",
+    "libp2p": "^0.23.1",
+    "libp2p-kad-dht": "^0.10.5",
+    "libp2p-mplex": "^0.8.2",
+    "libp2p-tcp": "^0.13.0",
+    "msgpack-lite": "^0.1.26",
+    "@holochain/n3h-common": "^0.0.1",
+    "@holochain/n3h-ipc": "^0.0.1",
+    "@holochain/tweetlog": "^0.0.1",
+    "@holochain/n3h-mod-message-libp2p": "^0.0.1",
+    "peer-id": "^0.11.0",
+    "peer-info": "^0.14.1",
+    "pull-stream": "^3.6.9",
+    "zeromq": "^5.1.0"
```

Comment:
Too many unused dependencies, I think this can probably be decreased to:

```json
     "@holochain/n3h-common": "^0.0.1",
     "@holochain/n3h-ipc": "^0.0.1",
     "@holochain/tweetlog": "^0.0.1",
```

Reply:
Done